<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserClass</class>
 <widget class="QWidget" name="UserClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User</string>
  </property>
  <widget class="QPushButton" name="addFriend_Button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>删除好友</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addComment_Button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>订阅评论</string>
   </property>
  </widget>
  <widget class="QLabel" name="usrname">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>邮箱：</string>
   </property>
  </widget>
  <widget class="QLabel" name="mail">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>加入时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="joinTime">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="user.qrc"/>
 </resources>
 <connections/>
</ui>
